packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -177,7 +177,10 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
 
     /// @inheritdoc IFastBridgeV2
     function bridge(BridgeParams memory params, BridgeParamsV2 memory paramsV2) public payable {
-        int256 exclusivityEndTime = int256(block.timestamp) + paramsV2.quoteExclusivitySeconds;
+        int256 exclusivityEndTime = paramsV2.quoteRelayer != address(0)
+            // prettier-ignore
+            ? int256(block.timestamp) + paramsV2.quoteExclusivitySeconds
+            : int256(0);
```

Comment:
I think this might a bit more clear without adding a gas overhead:
```suggestion
        int256 exclusivityEndTime = 0;
        if (paramsV2.quoteRelayer != address(0)) {
            exclusivityEndTime = int256(block.timestamp) + paramsV2.quoteExclusivitySeconds;
        }
```

Reply:
Also adding any kind of generic comment wrt `exclusivityEndTime` logic here would be good for easier understanding in the future (both for us and external reviewers).